src/angular/components/view.component.ts

```diff
@@ -184,6 +190,14 @@ export class ViewComponent implements OnDestroy, OnInit {
         event.dataTransfer.setData('text', data);
     }
 
+    public async minimizeIfNeeded(): Promise<void> {
```

Comment:
As-is, this is going into other Bitwarden apps like the browser extension and web vault. This should all be in the desktop repo since it is desktop specific. You can overload the copy function.

Reply:
Do you mean override the copy function ?